subsystem/elevator.py

```diff
@@ -0,0 +1,79 @@
+import rev
+
+from toolkit import Subsystem
+from toolkit.motors.rev_motors import SparkMax, SparkMaxConfig
+
+import config
+
+# TODO: CHANGE WHEN ROBOT IS BUILT
+ELEVATOR_CONFIG = SparkMaxConfig(
+    0.055, 0.0, 0.01, 0.000, (-.5, .75), idle_mode=rev.CANSparkMax.IdleMode.kBrake
+)
+
+class Elevator(Subsystem):
+
+    motor_extend: SparkMax = SparkMax(
+        config.elevator_can_id, config=ELEVATOR_CONFIG, inverted=False
+    )
+
+    def __init__(self) -> None:
+        super().__init__()
+        # Absolute encoder
+        self.zeroed: bool = False
+
+    def init(self) -> None:
+        self.motor_extend.init()
+        self.encoder = self.motor_extend.abs_encoder()
+        self.motor_extend.motor.setClosedLoopRampRate(config.elevator_ramp_rate)
+
+    def set_length(self, length: float) -> None:
+        """
+        Sets the length of the elevator in meters
+
+        Args:
+            length (meters): Length in meters
+        """
+        
+        current_length = self.get_length()
+
+        if length > current_length:
+            ff = config.elevator_feed_forward
+        elif length < current_length:
+            ff = -config.elevator_feed_forward
+        else: ff = 0
+        
+        self.motor_extend.pid_controller.setReference(length, rev.CANSparkMax.ControlType.kPosition, arbFeedforward=ff)
```

Comment:
Remember to multiply `length` by the ratio and also divide by the driver gear circumference. That would be a new variable in constants